Source/Eto/Drawing/PaddingConverter.cs

```diff
@@ -12,6 +12,11 @@ namespace Eto.Drawing
 	public class PaddingConverter : TypeConverter
 	{
 		/// <summary>
+		/// The character to split up the string which will be converted
+		/// </summary>
+		private readonly string[] StringSplitter = new string[1] { "," };
```

Comment:
I don't think this really matters. It is just one *test and conditional jump more* which is like nothing for the processor. It could even be `static char Separator = ','` and it would work (while new single-element array would be constructed each time unless they decide to optimize it by adding overload for split-by-sinle-char as they have it for format).

C# is more about code readability, about productivity. You'd have to use `IndexOf` and maybe `unsafe - fixed` to get better performance. Is it worth it?